package.json

```diff
@@ -11,6 +11,7 @@
     "lifecycle_conductor": "node extensions/lifecycle/conductor/service.js",
     "lifecycle_bucket_processor": "node extensions/lifecycle/bucketProcessor/task.js",
     "lifecycle_object_processor": "node extensions/lifecycle/objectProcessor/task.js",
```

Comment:
should we rename this to `lifecycle_object_expiration_processor` because the default is `expiration`?

Reply:
let's not break backward compatibility.